packages/graphql-modules/src/module/resolvers.ts

```diff
@@ -92,6 +92,18 @@ export function createResolvers(
                 resolvers[typeName][fieldName].resolve = resolver;
               }
 
+              if (isDefined((obj[fieldName] as any).extensions)) {
+                // some extensions allow to omit the resolve function, e.g. grafast
+                const defaultResolver = (val: any) => val;
+                const resolver = wrapResolver({
+                  config,
+                  resolver: (obj[fieldName] as any).resolve || defaultResolver,
+                  middlewareMap,
+                  path,
+                });
+                resolvers[typeName][fieldName].resolve = resolver;
+              }
+
```

Comment:
Is this needed? It doesn't seem to do anything with the extensions, and adding a resolver where one isn't needed unnecessarily increases memory usage of the schema (and will also break Grafast plans).

```suggestion
```